src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_params.py

```diff
@@ -60,6 +61,7 @@ def get_hostname_completion_list(prefix, action, parsed_args, **kwargs): # pylin
 register_cli_argument('appservice web create', 'name', options_list=('--name', '-n'), help='name of the new webapp')
 register_cli_argument('appservice web create', 'plan', options_list=('--plan', '-p'), completer=get_resource_name_completion_list('Microsoft.Web/serverFarms'),
                       help="name or resource id of the app service plan. Use 'appservice plan create' to get one")
+register_cli_argument('appservice web browse', 'start_trace', action='store_true', help='start live tracing')
```

Comment:
How about `--logstream -l`?  Also, the kudu endpoint name and terminology are `logstream`. 
I am a bit skeptical towards `logs` is we have 2 logging modes, one is the `stream` and the other is `zipped file log` which are both used. If possible I like to differentiate them clearly